ui/src/app/pages/workspace/create-billing-account-modal.tsx

```diff
@@ -1,76 +1,417 @@
+      [0, () => (<ModalFooter data-test-id='step-0-modal' style={{marginTop: 0, justifyContent: 'flex-start'}}>
+        <FlexRow style={{justifyContent: 'space-evenly'}}>
+      <FlexColumn>
+        <TextColumn>
+          <p style={styles.textHeader}>Familiar with setting up a Google Cloud Platform account?</p>
+          <p style={styles.textNormal}>Manually set up an account in <br/> Google Cloud Platform.</p>
+        </TextColumn>
+        <Button type='primary'
+                style={{marginTop: '0.5rem', fontWeight: 500, fontSize: '14px', height: '39px', width: '188px'}}
+                onClick={() => window.open(supportUrls.createBillingAccount, '_blank')}>
+          Read Directions
+        </Button>
+      </FlexColumn>
+      <FlexColumn>
+        <TextColumn>
+          <p style={styles.textHeader}>Let a Google billing partner create the account for you.</p>
+          <p style={styles.textNormal}>A representative will help you set up <br/>your billing account.</p>
+        </TextColumn>
+        <Button data-test-id='use-billing-partner-button'
+                type='primary'
+                style={{marginTop: '0.5rem', fontWeight: 500, fontSize: '14px', height: '39px', width: '220px'}}
+                onClick={() => {setCurrentStep(1); }}>
+          USE A BILLING PARTNER
+        </Button>
+       </FlexColumn>
+      </FlexRow>
+      </ModalFooter>)
+      ], [1, () => (<ModalFooter data-test-id='step-1-modal' style={{marginTop: 0, justifyContent: 'flex-start'}}>
+        <FlexColumn style={{justifyContent: 'space-evenly', width: '37rem'}}>
+          <div style={styles.textHeader}>Your Information</div>
+          <FlexRow style={{marginTop: '20px'}}>
+            <FlexColumn style={styles.textNormal}>
+              Your name
+              <TextInput
+                  data-test-id='user-full-name'
+                  style={styles.textInput}
+                  disabled={true}
+                  value={givenName + ' ' + familyName}/>
+            </FlexColumn>
+            <FlexColumn style={styles.textNormal}>
+              Your phone number
+              <TextInput
+                  data-test-id='user-phone-number'
+                  style={styles.textInput}
+                  onChange={(v) => validatePhoneNumber(v)}/>
+              {invalidPhoneNumberInput && <div data-test-id='invalidPhoneNumber' style={{color: colors.danger}}>
+                Invalid phone number input
+              </div>}
+            </FlexColumn>
+          </FlexRow>
+          <FlexRow style={{marginTop: '20px'}}>
+            <FlexColumn style={styles.textNormal}>
+              Your contact email address
+              <TextInput
+                  data-test-id='user-contact-email'
+                  disabled={true}
+                  style={styles.textInput}
+                  value={contactEmail}/>
+            </FlexColumn>
+            <FlexColumn style={styles.textNormal}>
+              Your Researchallofus ID
```

Comment:
again - I think think this is an issue in the mocks, I would say this should be `Your researchallofus.org ID`

Reply:
ah, I missed that comment